plugins/action/k8s_info.py

```diff
@@ -71,6 +75,118 @@ def run(self, tmp=None, task_vars=None):
         if src:
             new_module_args['src'] = src
 
+        template = self._task.args.get('template', None)
+        if template:
+            # template is only supported by k8s module.
+            if self._task.action not in ('k8s', 'community.kubernetes.k8s'):
```

Comment:
Would this have any impact on us using this feature in `community.okd.k8s`? Or would we just need to write our own action plugin there (if possible it would probably be nicer to reuse this one, as there probably wouldn't be any differences in functionality). Would it be possible to instead check the argspec to see if it includes a template parameter or something?

Reply:
I will include `community.okd.k8s` in this list so that the action plugin works with okd collection.

Reply:
action plugin is executed before the argspec check in the module, so it better to check here.